src/main/java/com/microsoft/graph/httpcore/ChaosHttpHandler.java

```diff
@@ -1,77 +0,0 @@
-package com.microsoft.graph.httpcore;
```

Comment:
keep this one

Reply:
disregard this comment, I just remembered it's been moved to kiota core https://github.com/microsoft/kiota-java/blob/main/components/http/okHttp/src/main/java/com/microsoft/kiota/http/middleware/ChaosHandler.java